ipyopenlayers/example.py

```diff
@@ -38,7 +48,27 @@ class Map(DOMWidget):
     value = Unicode('Hello World').tag(sync=True)
 
     layers = List(Instance(TileLayer)).tag(sync=True, **widget_serialization)
-
+    
     def add_layer(self, layer):
         # Copy layers (workaround ipywidgets issue)
         self.layers = self.layers + [layer]
+
+    def remove_layer(self, layer):
+        self.layers = [x for x in self.layers if x != layer]
+
+    def substitute(self, old, new):
+        if old.model_id not in self._layer_ids:
+            print("Could not substitute layer: layer not in layergroup.")
+        self.layers = tuple(
+            [new if layer.model_id == old.model_id else layer for layer in self.layers]
+        )
+
+
+    def clear_layer(self):
```

Comment:
Adding an "s" on the method name will make things clearer

```suggestion
    def clear_layers(self):
```